sympy/printing/tests/test_latex.py

```diff
@@ -2136,6 +2136,7 @@ def test_Pow():
 
 def test_issue_6975():
     x = symbols('x')
+    y = symbols('y')
```

Comment:
We can remove redundant definitions cause they are declared on the top.

Reply:
Since both x and y are declared as symbols (with no assumptions) at the top of the file these two lines can be removed.